### ISSUE TITLE
metrics

### ISSUE THREAD

**Issue by djdwui** [Jun-Jie-Shi/M2FTrans#12]
<img width="593" height="252" alt="Image" src="https://github.com/user-attachments/assets/77c35d15-7de8-49c2-be8f-e75636424c96" />Hello, Which checkpoint was used for the final results in the paper, model_best.pth or model_last.pth? Why is my ET score much lower than the result reported in the paper? Is this score calculated using ETPro Dice or ET Dice? Which part should I modify to make my results closer to the reported metrics?And, I see the batch size is set to 4 in the code, while it is 2 in the paper. Which one should I follow?
Thank you very much for your help!!

**Comment by Jun-Jie-Shi** [comment]
@djdwui Hello! During my experiments I used `model_last.pth` without performing validation, and reported the ETPro Dice score. The current version of the code appears to have modified the masks for different batches during training, whereas the version used in previous experiments applied the same mask to every batch of samples. As it has been some time, I am not certain whether this was the cause of the issue. Regarding the batch size, I think `batchsize=2` was used on each graphics card. I suggest you use whichever settings you prefer, provided you maintain consistency across all experiments. Alternatively, you could use the pre-trained weights provided in our README for validation. I hope this helps!

**Comment by djdwui** [comment]
thank you so much!!
